TAF/utils/scripts/docker/arm64.env

```diff
@@ -1,5 +1,10 @@
+# Image Variables
 virtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-go-arm64:master
 mqtt=nexus3.edgexfoundry.org:10004/docker-device-mqtt-go-arm64:master
 random=nexus3.edgexfoundry.org:10004/docker-device-random-go-arm64:master
 modbus=nexus3.edgexfoundry.org:10004/docker-device-modbus-go-arm64:master
 modbusSimulator=panterdsd/diagslave
+appService=nexus3.edgexfoundry.org:10004/docker-app-service-configurable-arm64:master
+
+# Additional Variables
+DOCKER_HOST_IP=172.17.0.1
```

Comment:
if the environment variable is common, please move to a single file, such as "common.env"

Reply:
Move DOCKER_HOST_IP to common.env file.